src/main/scala/xiangshan/cache/mmu/L2TLB.scala

```diff
@@ -302,6 +350,9 @@ class L2TLBImp(outer: L2TLB)(implicit p: Parameters) extends PtwModule(outer) wi
   def from_hptw(id: UInt) = {
     id === l2tlbParams.llptwsize.U + 1.U
   }
+  def from_bitmap(id: UInt) = {
+    id > l2tlbParams.llptwsize.U + 1.U
```

Comment:
Is it convenient to set a cap here to prevent future misjudgements, for example `id > l2tlbParams.llptwsize.U + 1.U && id <= xxx`